recipes/glimmerhmm/build.sh

```diff
@@ -5,16 +5,21 @@ if [ `uname` == Darwin ]; then
         sed -i.bak "s~#include <malloc.h>~#include <malloc/malloc.h>~g" train/utils.c
 fi
 
-
 mkdir -p $PREFIX/bin
 mkdir -p $PREFIX/GlimmerHMM/trained_dir
 
 # cd into the sources
-cd sources && make
-cd ../train && make
-cd ..
+if [ `uname` == Darwin ]; then
+        cd sources && make CC=$CXX
+        cd ../train && make C=$C CC=$CXX
```

Comment:
I this really using a C compiler? If so you need to add it in the build-dependencies.